spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/saml2/Saml2RelyingPartyProperties.java

```diff
@@ -50,13 +50,26 @@
 	 */
 	public static class Registration {
 
+		/**
+		 * Relying Party identifier URI pattern.
```

Comment:
It would be good to base this description on how `RelyingPartyRegistration.Builder` describes it. Perhaps something like the following:

```
Local relying party or Service Provider entity ID template. Can generate its entity ID based on possible variables of baseUrl, registrationId, baseScheme, baseHost, and basePort. For example, {baseUrl}/saml2/service-provider-metadata/{registrationId}.
```

The field javadoc is used in the configuration property metadata so needs to avoid using things like `{@code}`.

Reply:
Using the description from `RelyingPartyRegistration.Builder#entityId`, with the addition of the default value:

     Relying party's EntityID. This value may contain a number of placeholders. They are: baseUrl, registrationId, baseScheme, baseHost, and basePort. The default value is {baseUrl}/saml2/service-provider-metadata/{registrationId}.